imports/plugins/core/taxes/server/methods/methods.js

```diff
@@ -124,6 +125,36 @@ export const methods = {
     });
   },
 
+  /**
+   * @name "taxes/updateTaxCode"
+   * @method
+   * @memberof Methods/Taxes
+   * @summary updates the taxcode on options of a product.
+   * @param  {String} products array of products to be updated.
+   * @return {undefined} returns nothing
```

Comment:
I think it's best practice to always have a function return a value rather than undefined. Maybe a count of updated products or whatever